### PR TITLE
Broken bones can now cause IB

### PR DESCRIPTION
**Pull request by BeagleGaming1**
# About the pull request

Unsplinted broken bones can cause IB in a similar way to organ damage

# Explain why it's good for the game

It only makes sense that loose bones in your limbs will cut you up.

# Changelog

:cl:
balance: Unsplinted broken bones that are not hands or feet can now cause IB
balance: bloodbag capacity raised to 300u
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/life/handle_organs.dm [hunk 1]
```diff
@@ -25,6 +25,11 @@
 					var/damage = rand(3,5)
 					I.take_damage(damage)
 					pain.apply_pain(damage * PAIN_ORGAN_DAMAGE_MULTIPLIER)
+				if(E.is_broken() && prob(15))
```
Comment (morrowwolf) [inline code/modules/mob/living/carbon/human/life/handle_organs.dm:28]: Change prob to like 3

**Review by morrowwolf — CHANGES_REQUESTED** [review]

**Review by morrowwolf — APPROVED** [review]
code and balance approved

**Comment by MistChristmas** [comment]
If this is here to stay might we add IBs visible to the normal health scanner. So that we can see when someone has 5 IBs and where. Cause playing the guessing game of which frac(s) have the IB(s) isn't exactly a fun time.

**Comment by BeagleGaming1** [comment]
> If this is here to stay might we add IBs visible to the normal health scanner. So that we can see when someone has 5 IBs and where. Cause playing the guessing game of which frac(s) have the IB(s) isn't exactly a fun time.

Right click, "Check Status", it shows you exactly which limb has the IB

**Review by morrowwolf — CHANGES_REQUESTED** [review]
Let's do non-hands/non-feet only and drop prob to 2.

**Pull request merged**